src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.io.File;
+
+public class Main {
+    public static void main(String[] args) {
+        String [] files = new String[] {
+                "apple.csv",
+                "banana.csv",
+                "grape.csv",
+                "orange.csv"
+        };
+        String [] resultFiles = new String[]{
+                "apple_result.txt",
+                "banana_result.txt",
+                "grape_result.txt",
+                "orange_result.txt"
+        };
+
+        WorkWithFile workWithFile = new WorkWithFile();
```

Comment:
The 'WorkWithFile' class instance 'workWithFile' is currently using the default access modifier. If this class is only intended to be used within the same package, the access modifier is appropriate. If it's meant to be used more broadly, consider making it public.